index.bs

```diff
@@ -2854,61 +2914,114 @@ Instances of {{WritableStream}} are created with the internal slots described in
-      or failure.
-
-    <li><p><code>write(chunk, controller)</code> is called when a new <a>chunk</a> of data is ready to be written to the
-      <a>underlying sink</a>. It can return a promise to signal success or failure of the write operation. The stream
-      implementation guarantees that this method will be called only after previous writes have succeeded, and never
-      after <code>close</code> or <code>abort</code> is called.
-
-    <li><p><code>close()</code> is called after the producer signals that they are done writing chunks to the
-      stream, and all queued-up writes successfully complete. It can perform any actions necessary to finalize writes
-      to the <a>underlying sink</a>, and release access to it. If this process is asynchronous, it can return a promise
-      to signal success or failure. The stream implementation guarantees that this method will be called only after all
-      queued-up writes have succeeded.
-
-    <li><p><code>abort(reason)</code> is called when the producer signals they wish to abruptly close the stream
-      and put it in an errored state. It can clean up any held resources, much like <code>close</code>, but perhaps
-      with some custom handling. Unlike <code>close</code>, <code>abort</code> will be called even if writes are queued
-      up; those <a>chunks</a> will be thrown away. If this process is asynchronous, it can return a promise to signal
-      success or failure.
-  </ul>
-
-  The <code>controller</code> object passed to <code>start</code>, <code>write</code> and <code>close</code> is an
-  instance of {{WritableStreamDefaultController}}, and has the ability to error the stream.
-
-  The constructor also accepts a second argument containing the <a>queuing strategy</a> object with
-  two properties: a non-negative number <code>highWaterMark</code>, and a function <code>size(chunk)</code>. The
-  supplied <code>strategy</code> could be an instance of the built-in {{CountQueuingStrategy}} or
-  {{ByteLengthQueuingStrategy}} classes, or it could be custom. If no strategy is supplied, the default
-  behavior will be the same as a {{CountQueuingStrategy}} with a <a>high water mark</a> of 1.
+  The <code>strategy</code> argument represents the stream's <a>queuing strategy</a>, as described in [[#qs-api]]. If it
+  is not provided, the default behavior will be the same as a {{CountQueuingStrategy}} with a <a>high water mark</a>
+  of 1.
 </div>
 
 <emu-alg>
   1. Perform ! InitializeWritableStream(_this_).
   1. Let _type_ be ? GetV(_underlyingSink_, `"type"`).
   1. If _type_ is not *undefined*, throw a *RangeError* exception. <p class="note">This is to allow us to add new
      potential types in the future, without backward-compatibility concerns.</p>
+  1. Let _size_ be ? GetV(_readableStrategy_, `"size"`).
   1. Let _sizeAlgorithm_ be ? MakeSizeAlgorithmFromSizeFunction(_size_).
+  1. Let _highWaterMark_ be ? GetV(_readableStrategy_, `"highWaterMark"`).
```

Comment:
Because MakeSizeAlgorithmFromSizeFunction() will throw if size is not a function, placing this GetV() after that operation changes behaviour.